scripts/generator/graphql_generator/csharp/SDK/DesktopMessageReceiver.cs.erb

```diff
@@ -0,0 +1,56 @@
+namespace <%= namespace %>.SDK {
+    using System;
+    using Shopify.Unity.SDK;
+    using Shopify.Unity.GraphQL;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    public partial class DesktopMessageReceiver: MonoBehaviour {}
+    #endif
+
+    public partial class DesktopMessageReceiver {
```

Comment:
I agree on this too. It might be worthwhile to explore a delegate pattern where the `Receiver` is just basically a listener for when the Web browser is closed. Ideally the handling of verification and callbacks should be handled elsewhere not in the receiver class.